tests/deb_rpm_tar/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Just drop this test. If we want to test for a single file, then we should be doing an analysis test.

Reply:
Done